cypress_shared/pages/temporary-accommodation/manage/lostBedEditable.ts

```diff
@@ -0,0 +1,30 @@
+import Page from '../../page'
+import errorLookups from '../../../../server/i18n/en/errors.json'
+import { NewLostBed } from '../../../../server/@types/shared/models/NewLostBed'
+
+export default abstract class LostBedEditablePage extends Page {
+  shouldShowDateConflictErrorMessages(): void {
+    ;['startDate', 'endDate'].forEach(field => {
+      cy.get('.govuk-error-summary').should('contain', errorLookups.generic[field].conflict)
+      cy.get(`[data-cy-error-${field}]`).should('contain', errorLookups.generic[field].conflict)
+    })
+  }
+
+  protected completeEditableForm(newOrUpdateLostBed: NewLostBed, roomId: string): void {
+    this.getHiddenInputByNameAndSetValue('bedId', roomId)
```

Comment:
I feel like in principle this shouldn't be necessary? Everything on the page is coming from our own faked API, so we should be able control the contents of the hidden for via that?

Reply:
hmmm yeah, not sure what I was trying to achieve there. I guess that was when I was confused about bedIds and roomIds. Removed now anyway 👍